src/commands/start-client.js

```diff
@@ -41,7 +44,7 @@ module.exports = function () {
     // Proxy http requests from server to client in development mode
     app.use(proxy({
       changeOrigin: false,
-      target: "http://localhost:8880",
+      target: `${server.protocol}://${server.host}:${appConfig.assetPort}`,
```

Comment:
same as above, appConfig.assetPort should fall back to `8880` so we don't break backwards compatibility on older generated apps